exercises/parallel-letter-frequency/benches/benchmark.rs

```diff
@@ -0,0 +1,73 @@
+#![feature(test)]
+extern crate parallel_letter_frequency;
+extern crate test;
+
+use std::collections::HashMap;
+use test::Bencher;
+
+#[bench]
+fn bench_sequential(b: &mut Bencher) {
+    let texts = all_texts();
+    b.iter(|| frequency(&texts));
+}
+
+#[bench]
+fn bench_parallel(b: &mut Bencher) {
+    let texts = all_texts();
+    b.iter(|| parallel_letter_frequency::frequency(&texts, 3));
+}
+
+/// Simple sequential char frequency. Can it be beat?
+pub fn frequency(texts: &[&str]) -> HashMap<char, usize> {
```

Comment:
I'm in favor of leaving this visible. I personally don't think this exercise should focus too much on the details of getting the count right and more on concurrency techniques.

In other words, reading this doesn't give away anything about threads or whatever tools you'd use to solve this.